qiita_pet/templates/list_studies.html

```diff
@@ -0,0 +1,266 @@
+{% extends sitebase.html %}
+{% block head %}
+<link rel="stylesheet" href="/static/vendor/css/jquery.dataTables.css" type="text/css">
+
+<script src="/static/vendor/js/jquery.dataTables.min.js"></script>
+<script src="/static/vendor/js/jquery.dataTables.plugin.natural.js"></script>
+
+<script type="text/javascript">
+var current_study;
+var ajaxURL = "/study/search/?&user={{current_user.id}}&sEcho=" + Math.floor(Math.random()*1001);
```

Comment:
If the goal is to reduce the possibility of attack, and if this is a vulnerability, then I recommend using a cryptographically secure PRNG. Details [here](http://stackoverflow.com/questions/5651789/is-math-random-cryptographically-secure).